migrations/sql/V2018.02.15.1029__ChangeBurdenEstimateColumnDataTypes.sql

```diff
@@ -0,0 +1,27 @@
+-- this ran out of disk space on UAT
```

Comment:
what's going on with this file?

Reply:
This file contains 2 ways of doing things, commented out. I was testing them both to compare performance but couldn't actually run either of them on UAT.